# Assignment 1_Parametric tests

BACKGROUND:
In a randomized control trial, 32 patients were divided into two groups: A and B. Group A received test drug whereas group B received placebo. The variable of interest was ‘Change in pain level’ measured by visual analogue scale (VAS)’ before treatment and after 3 days of treatment.

In [ ]:
# Import VAS DATA and name it as pain_vas. Check for normality of the data

library(readr)
library(dplyr)

pain_vas <- read_csv("/content/VAS DATA.csv")

# Check for normality
group_a <- pain_vas %>% filter(Group == 'A')
group_b <- pain_vas %>% filter(Group == 'B')

# Shapiro-Wilk test for normality
shapiro.test(group_a$VAS_before)
shapiro.test(group_a$VAS_after)
shapiro.test(group_b$VAS_before)
shapiro.test(group_b$VAS_after)

Rows: 32 Columns: 3
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (1): Group
dbl (2): VAS_before, VAS_after

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.



	Shapiro-Wilk normality test

data:  group_a$VAS_before
W = 0.95239, p-value = 0.5284



	Shapiro-Wilk normality test

data:  group_a$VAS_after
W = 0.87669, p-value = 0.03448



	Shapiro-Wilk normality test

data:  group_b$VAS_before
W = 0.98225, p-value = 0.9789



	Shapiro-Wilk normality test

data:  group_b$VAS_after
W = 0.968, p-value = 0.8053


The results of the Shapiro-Wilk test for normality are:

Group A VAS_before:
p=0.528 (normal distribution)

Group A VAS_after:
p=0.034 (non-normal distribution)

Group B VAS_before:
p=0.979 (normal distribution)

Group B VAS_after:
p=0.805 (normal distribution)


In [ ]:
# Is post treatment VAS score significantly less as compared to ‘before treatment’ VAS score for Group A?

# Since Group A's post-treatment VAS score is non-normally distributed, we use the Wilcoxon Signed-Rank Test
wilcox.test(group_a$VAS_before, group_a$VAS_after, paired = TRUE)

Warning message in wilcox.test.default(group_a$VAS_before, group_a$VAS_after, paired = TRUE):
“cannot compute exact p-value with ties”



	Wilcoxon signed rank test with continuity correction

data:  group_a$VAS_before and group_a$VAS_after
V = 136, p-value = 0.0004782
alternative hypothesis: true location shift is not equal to 0


p = 0.0004782 indicating a significant reduction in VAS score after treatment

In [ ]:
#  Is Post Treatment VAS Score Significantly Less Compared to Pre-treatment for Group B?

# For Group B, both pre-treatment and post-treatment VAS scores are normally distributed, so we can use a Paired T-test.
t.test(group_b$VAS_before, group_b$VAS_after, paired = TRUE)



	Paired t-test

data:  group_b$VAS_before and group_b$VAS_after
t = 2.4252, df = 15, p-value = 0.02839
alternative hypothesis: true mean difference is not equal to 0
95 percent confidence interval:
 0.196836 3.053164
sample estimates:
mean difference 
          1.625 


p = 0.02839 indicating a significant reduction in VAS score after treatment

In [ ]:
# Is the Average Change in Pain Level for Group A Significantly More than Group B?

# Calculate the change in VAS
group_a$change_in_VAS <- group_a$VAS_before - group_a$VAS_after
group_b$change_in_VAS <- group_b$VAS_before - group_b$VAS_after

# Check normality using the Shapiro-Wilk test
shapiro_a_change <- shapiro.test(group_a$change_in_VAS)
shapiro_b_change <- shapiro.test(group_b$change_in_VAS)

shapiro_a_change
shapiro_b_change


	Shapiro-Wilk normality test

data:  group_a$change_in_VAS
W = 0.94923, p-value = 0.4776



	Shapiro-Wilk normality test

data:  group_b$change_in_VAS
W = 0.92594, p-value = 0.2101


The changes in VAS scores for both Group A and Group B follow a normal distribution.

In [ ]:
t_test_result <- t.test(group_a$change_in_VAS, group_b$change_in_VAS, var.equal = FALSE, alternative = "greater")
t_test_result


	Welch Two Sample t-test

data:  group_a$change_in_VAS and group_b$change_in_VAS
t = 11.121, df = 16.728, p-value = 1.903e-09
alternative hypothesis: true difference in means is greater than 0
95 percent confidence interval:
 26.88412      Inf
sample estimates:
mean of x mean of y 
   33.500     1.625 


This suggests that the mean change in pain level for Group A is significantly greater than for Group B.

In other words, the treatment given to Group A is far more effective in reducing pain compared to the treatment received by Group B.